### PR TITLE
feat(docs): add 2019Q4 roadmap

### PR DESCRIPTION
**Pull request by dhmlau**
Roadmap for 2019Q4

@raymondfeng @bajtos, this is based on the box note that we were working together.  Please feel free to update this PR. Thanks. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -1,3 +1,100 @@
...
+- Implement TestSandbox.writeTextFile https://github.com/strongloop/loopback-next/issues/3731
+- Improve `defineCrudRestController` to create a named controller class https://github.com/strongloop/loopback-next/issues/3732
+- Add `defineCrudRepositoryClass` - a helper to create a named repository class https://github.com/strongloop/loopback-next/issues/3733
+- Model API booter & builder https://github.com/strongloop/loopback-next/issues/3736
+- Add CrudRestApiBuilder to `@loopback/rest-crud` https://github.com/strongloop/loopback-next/issues/3737
+- Example app showing CrudRestApiBuilder https://github.com/strongloop/loopback-next/issues/3738
+
+Stretch goals:
+- From relation definition to REST API with auto-generated repository/controller classes https://github.com/strongloop/loopback-next/issues/2483
+- From datasource config to Service REST API with no proxy/controller classes https://github.com/strongloop/loopback-next/issues/3717
```
Comment (bajtos) [inline docs/ROADMAP.md]: > Stretch goals:
> - From relation definition to REST API with auto-generated repository/controller classes >
> - From datasource config to Service REST API with no proxy/controller classes 

Considering how much work we have already added to the Q4 roadmap, I am proposing to remove these two stretch goals. I feel the other items are more important, we should keep some buffer to add new stories discovering along the way, plus it's probably better to wait for feedback on the initial ModelApiBuilder version before extending it with additional features.

@dhmlau @raymondfeng thoughts?

Comment (dhmlau) [reply docs/ROADMAP.md]: +1
We might want to leave some room for making our migration story better.  
Actually would these 2 be hindering the LB3->LB4 migration?

**Comment by dhmlau** [comment]
@raymondfeng @bajtos, in my latest commit, I've removed the tasks that we managed to complete in Q3.  Could you please review it again?  Thanks!

**Comment by dhmlau** [comment]
Talked to @raymondfeng yesterday, he's good with the proposal.

**Pull request merged**